ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-504] Add trust fund page

## Description of change
Adds trust fund page to capital flow. Also displays in review application section

## Link to relevant ticket
[CRIMAPP-585](https://dsdmoj.atlassian.net/browse/CRIMAPP-585)

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:

### After changes:

**Form validation**
<img width="775" alt="Screenshot 2024-03-12 at 09 29 51" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/0d3cf015-5b94-442c-a764-fbecbdbbe16e">

**Scenario when yes selected and data entered**
<img width="875" alt="Screenshot 2024-03-12 at 08 19 33" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/fd849550-6637-45d6-acc6-3063f6c8dec1">
<img width="787" alt="Screenshot 2024-03-12 at 09 30 49" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/abd9734e-7940-4b0a-85f7-b9c1c4477d8b">

**Scenario when no selected** 
<img width="699" alt="Screenshot 2024-03-12 at 09 27 16" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/effe98fc-e670-4895-bac7-5cf070b5c915">
<img width="728" alt="Screenshot 2024-03-12 at 09 29 36" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/c1440e76-2250-4363-ae72-194ec3e3d215">

## How to manually test the feature

Start an application and fill in the details till you get to the trust fund page in the capital assessment section
Select yes and validate the additional input fields for trust fund amount held and yearly dividend are displayed 
Then select save and continue and confirm the validation errors are displayed
Enter data into the additional fields and confirm you can save and continue to the upload evidence screen
Continue with the application to the review page and confirm you can see a trust funds section with the data matching what you entered
Check the change links take you back to the trust fund page 
Select no and save and continue 
Confirm that the review application screen displays no as the selected answer for the trust fund question

[CRIMAPP-585]: https://dsdmoj.atlassian.net/browse/CRIMAPP-585?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
